src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = !century
+    ? people.filter(({ sex }) => sex === 'm')
+    : people.filter(({ sex, died }) => sex === 'm'
+    && Math.ceil(died / 100) === century);
+
```

Comment:
You have two filter calls - but a cleaner option would be a single filter call with more complex predicate